scripts/check_frontend_coverage.py

```diff
@@ -335,6 +335,7 @@
     'suggestion-modal-for-exploration-player.service.ts',
     'summary-list-header.directive.ts',
     'supplemental-card.directive.ts',
+    'svg-sanitizer.service.ts',
```

Comment:
Why is this added? See the note at the top. Did you already get approval from @nithusha21?

(Also @nithusha21 should we change this to @U8NWXD?)

Reply:
Hmm, I don't know why its being added. If something is not possible to be covered in that file, let me know. If not can we try to get 100% coverage on the newly added files?

Also, I think I can move this to @U8NWXD, will talk to him about it!

Reply:
IIRC, this from my PR #11306. This is a new file, but the content it has is actually from another file in the list (image-uploader-helper.service). I moved some logic from that file and that was not covered previously.

Reply:
Yes, I can confirm this is a part of #11306.
See https://github.com/oppia/oppia/pull/11850#issuecomment-786942336 for details.